proto/cometbft/abci/v1/types.proto

```diff
@@ -436,6 +437,12 @@ message FinalizeBlockResponse {
   // that execution of the transactions was deterministic.
   // It is up to the application to decide which algorithm to use.
   bytes app_hash = 5;
+  // delay between the time when this block is committed and the next height is started.
```

Comment:
@andynog and I had a discussion last night about how we are modifying protobufs on `main` and/or `v1.x`. We have added a point to discuss this in next week's meeting, as it is complex and we'd like everyone in the team to be aware of it, and provide feedback.
If you want to know what this is all about, take a look at #3090.
This shouldn't be blocking this PR, it's just to raise awareness.